### PR TITLE
MongoDB backup Amazon S3

### PR DESCRIPTION
**Pull request by felipe-augusto**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [X] I have tested the template using the method described in README.md thoroughly
- [X] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [X] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [X] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [X] Icon is added as a png file to the logos directory.

### CODE REVIEW

**Comment by felipe-augusto** [comment]
the docker image only has the [`latest` tag](https://hub.docker.com/r/halvves/mongodb-backup-s3/tags), maybe should try using the digest?

**Comment by githubsaturn** [comment]
Yes, please use a sha tag.

File: public/v2/apps/mongodb-backup-s3.json [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            "id": "$$cap_backup_folder",
+            "defaultValue": "mongo_dumps/",
+            "label": "BACKUP_FOLDER"
+        },
+        {
+            "description": "The host/ip of your mongodb database",
+            "id": "$$cap_mongodb_host",
+            "label": "MONGODB HOST",
+            "validRegex": "/^.+$/",
+            "defaultValue": "localhost"
```
Comment (githubsaturn) [inline public/v2/apps/mongodb-backup-s3.json:67]: This default value makes is easy for the user to just click on deploy and miss the fact that this is not right. Maybe remove the default value here, and just add some bit to the description. Like:
```
            "description": "The host/ip of your mongodb database. For CapRover hosted database, use srv-captain--your-mongodb-name",
```

File: public/v2/apps/mongodb-backup-s3.json [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+            "label": "EXTRA_OPTS"
+        },
+        {
+            "description": "The interval of cron job to run mongodump. 0 3 * * * by default, which is every day at 03:00hrs.",
+            "id": "$$cap_cron_time",
+            "label": "CRON_TIME",
+            "defaultValue": "0 3 * * *"
+        },
+        {
+            "description": "timezone",
```
Comment (githubsaturn) [inline public/v2/apps/mongodb-backup-s3.json:104]: What's the difference between timezone and cron timezone? Is there any specific reason behind making them two separate fields? Can we not just use one variable for both?

Comment (felipe-augusto) [reply public/v2/apps/mongodb-backup-s3.json:103]: It is not clear from the docker image [docs](https://github.com/halvves/mongodb-backup-s3):

```sh
TZ - timezone. default: US/Eastern

CRON_TZ - cron timezone. default: US/Eastern
```

I guess we can use the same variable

File: public/v2/apps/mongodb-backup-s3.json (hunk 1 above)
Comment (felipe-augusto) [reply public/v2/apps/mongodb-backup-s3.json]: done

**Comment by githubsaturn** [comment]
Thanks!

**Pull request merged**